scripts/src/main/resources/scripts/command/cobigen

```diff
@@ -32,13 +32,13 @@ function doSetup() {
 # $@: optional CobiGen arguments
 function doRunCLI() {
   local cobigen_cmd="cobigen"
-  doEcho "Running: ${cobigen_cmd} ${@}"
+  doEcho "Running: ${cobigen_cmd} ${*}"
   if doIsQuiet
   then
-    "${cobigen_cmd}" "${@}"
+    "${cobigen_cmd}" "${*}"
```

Comment:
The use of "${@}" vs "${*}" is a bit hard to understand in bash.  In this case,  you should **keep** the  "${@}", since that keeps the "quoting per each single argument", so  if you call this commandlet as
   commandlet   a  "start end"
these - hypothecial - arguments are correctly forwarded to the internally called command 
arg1=a
arg2="start end"